setup.py

```diff
@@ -1,5 +1,19 @@
-from setuptools import setup, find_packages
+from setuptools import setup, find_packages, Extension
 
+extensions = [Extension(
+        "fastdtw",
+        ["fastdtw.pyx"],
+        language="c++",
+        include_dirs=[],
+        libraries=["stdc++"],
+        extra_link_args=['-std=c++11'],
+        extra_compile_args=['-std=c++11'],
+    )]
+
+for e in extensions:
+    e.cython_directives = {'boundscheck': False,
+                           'cdivision': True,
+                           'wraparound': False}
```

Comment:
These directives are redundant here. They were specified in the Cython file, which is preferred.